### PR TITLE
Added advanced descriptions

### PR DESCRIPTION
**Pull request by arcbtc**
Needed for https://github.com/lnbits/lnbits/pull/2544

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -167,6 +171,18 @@
             "archive": "https://github.com/lnbits/tpos/archive/refs/tags/v0.7.1.zip",
             "hash": "559670df0055af8d99419910739c8906ad9b1bf41d88660452990d33e0dd17c5"
         },
+        {
+            "id": "tpos",
+            "repo": "https://github.com/lnbits/tpos",
+            "name": "TPoS",
+            "version": "0.7.2",
```
Comment (motorina0) [inline extensions.json:178]: there is no `0.7.2` release for TPOS

File: extensions.json [hunk 2]
```diff
@@ -957,10 +991,11 @@
             "id": "watchonly",
             "repo": "https://github.com/lnbits/watchonly",
             "name": "Onchain Wallet",
-            "version": "0.2.2",
+            "version": "0.2.0",
```
Comment (motorina0) [inline extensions.json]: why revert version?

File: extensions.json (hunk 1 above)
Comment (arcbtc) [reply extensions.json]: I know, weird, did you add?

File: extensions.json (hunk 2 above)
Comment (arcbtc) [reply extensions.json]: I didnt, wondered if you did?

Comment (arcbtc) [reply extensions.json]: fixed now

File: extensions.json (hunk 1 above)
Comment (arcbtc) [reply extensions.json]: fixed now

Comment (arcbtc) [reply extensions.json]: https://github.com/lnbits/lnbits-extensions/pull/374/commits/f3590764f767f6da62f41baf317b32195bc1cdc1

File: extensions.json (hunk 2 above)
Comment (arcbtc) [reply extensions.json]: https://github.com/lnbits/lnbits-extensions/pull/374/commits/32199084e1aa9e0325e8e683fe16e537f9fad0b3

**Review by dni — APPROVED** [review]

**Review by motorina0 — APPROVED** [review]

**Pull request merged**